svgelements/svgelements.py

```diff
@@ -5721,7 +5722,7 @@ def first_point(self):
         with a Move command with a None start in which case first point is that Move's destination."""
         if len(self._segments) == 0:
             return None
-        if self._segments[0].start is not None:
+        if not isinstance(self._segments[0], (Move,Close)) and self._segments[0].start is not None:
             return Point(self._segments[0].start)
```

Comment:
The first point in a path should be a move. And in a move you should give the start position if its not None else give the end position. I forget what exactly I was using this for but it was supposed to give me the origin of the move if that origin of the move existed. --- I'm worried about side effects here, and it seems a bit of breaking change. 

"unless it starts with a move command with a none-start" is exactly what the old code was doing.